src/web/releases.rs

```diff
@@ -398,14 +398,16 @@ pub fn author_handler(req: &mut Request) -> IronResult<Response> {
 
     let conn = extension!(req, Pool).get();
     let author = ctry!(router.find("author")
-        .ok_or(IronError::new(Nope::CrateNotFound, status::NotFound)));
-
-    let (author_name, packages) = if author.starts_with("@") {
-        let mut author = author.clone().split("@");
-        get_releases_by_owner(&conn,
-                              page_number,
-                              RELEASES_IN_RELEASES,
-                              cexpect!(author.nth(1)))
+        .ok_or_else(|| IronError::new(Nope::CrateNotFound, status::NotFound)));
```

Comment:
Is this change necessary? This isn't doing anything expensive and it adds mental overhead.

Reply:
It's actually a performance thing, `ok_or` always allocates, no matter the status of what it's called on, while `ok_or_else` is lazy, which means it's only allocated for when it's actually needed

Reply:
Right, but there's no allocation here. `IronError` is only on the stack, as is `Nope` and `NotFound`.

Reply:
Alright, I'll revert that